<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCharacterDialog</class>
 <widget class="QDialog" name="NewCharacterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="0,1">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="characterNameLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="characterNameLabel">
        <property name="text">
         <string>Character Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="characterNameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="groupsLayout">
      <item>
       <widget class="QGroupBox" name="frameSizeGroupBox">
        <property name="title">
         <string>Frame Size</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="frameSizeLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="frameWidthLabel">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="frameWidthBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="frameHeightLabel">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="frameHeightBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="imageOptGroupBox">
        <property name="title">
         <string>Image Optimization</string>
        </property>
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>142</width>
           <height>136</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="imageOptLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="marginXLabel">
            <property name="text">
             <string>Margin X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="marginYLavel">
            <property name="text">
             <string>Margin Y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="imageOptSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="xSpacingLabel">
            <property name="text">
             <string>X Spacing</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="ySpacingLabel">
            <property name="text">
             <string>Y Spacing</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="marginXBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="marginYBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="xSpacingBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="ySpacingBox">
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>characterNameLabel</zorder>
  <zorder>characterNameLabel</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewCharacterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewCharacterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
